examples/favorites_advanced/bloc_favorites_advanced/lib/feature_puppy/details/blocs/puppy_details_state.dart

```diff
@@ -0,0 +1,45 @@
+part of 'puppy_details_bloc.dart';
+
+@immutable
+class PuppyDetailsState extends Equatable {
```

Comment:
@StanislavPrime, what is the reason for all properties being nullable? It looks like this causes a big impact on using that state in the UI layer, as every time it needs to be added a nullability check.

Reply:
I removed the nullable properties and left only a non-nullable `Puppy `property in the `PuppyDetailsState`